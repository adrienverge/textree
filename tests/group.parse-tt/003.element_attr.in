element1(attr=value)
element2(attr=value with spaces)
element3(attr1=value1,attr2=value2)
element4(attr1 = value1 with padding  ,  attr2   =   other padded  value  )
element5(
  attr1 = value1
  attr2 = value2
)
