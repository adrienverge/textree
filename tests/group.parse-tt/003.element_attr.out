{"type":"start","name":"element1","attributes":{}}
{"type":"attr","name":"attr","value":"value"}
{"type":"end"}
{"type":"start","name":"element2","attributes":{}}
{"type":"attr","name":"attr","value":"value with spaces"}
{"type":"end"}
{"type":"start","name":"element3","attributes":{}}
{"type":"attr","name":"attr1","value":"value1"}
{"type":"attr","name":"attr2","value":"value2"}
{"type":"end"}
{"type":"start","name":"element4","attributes":{}}
{"type":"attr","name":"attr1","value":"value1 with padding"}
{"type":"attr","name":"attr2","value":"other padded  value"}
{"type":"end"}
{"type":"start","name":"element5","attributes":{}}
{"type":"attr","name":"attr1","value":"value1"}
{"type":"attr","name":"attr2","value":"value2"}
{"type":"end"}
